### DISCUSSION TITLE
Better self install instructions?

### DISCUSSION THREAD

**Discussion by holtmeister** [rnadigital/agentcloud#10] [Ideas]
I'm presuming the purpose of this app is to let non-developers use the power of autogen without knowing how to code. Given that, would it be possible to expand the self install instructions for non-tech people? Perhaps some ideas on how to self install using no code solutions?

**Comment by ragyabraham** [comment]
Hey @holtmeister i absolutely agree. And we're hoping that the cloud version will fill that void. For now, like most open source projects, there's a small amount of technical knowledge required to get the app started. However, the purpose of this app, is that once it's started that's where the technical requirements ends.

**Comment by kattapug** [comment]
Agree suggest docs to help self-hosted or local install for non-developers or newbies.
